src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.security.oauth2.resourceserver.jwt.issuer-uri
+spring.security.oauth2.authorizationserver.endpoint.jwk-set-uri=
```

Comment:
Did you check these properties, do they really exists?

Reply:
Removed the unnecessary one (the second property)